opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/instrumentor.py

```diff
@@ -24,7 +24,17 @@
 
 
 class BaseInstrumentor(ABC):
-    """An ABC for instrumentors"""
+    """An ABC for instrumentors
+
+    Child classes of this ABC are implemented to instrument specific third
```

Comment:
```suggestion
    Child classes of this ABC should instrument specific third
```
I'd suggest this wording change, as it's not necessarily true that are all child classes are implemented properly.